qemu/tests/nested_system_reset.py

```diff
@@ -0,0 +1,40 @@
+from virttest import error_context
+from virttest import utils_package
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    System_reset in nested environment
+
+    1. Boot L1 guest
+    2. Start a qemu-kvm process in L1 guest
+    3. Do system_reset for L1
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+
+    sm = utils_package.RemotePackageMgr(session, "qemu-kvm")
+    if not (sm.is_installed("qemu-kvm") or sm.install()):
+        test.fail("qemu-kvm is not installed in the guest vm.")
```

Comment:
Can be simplified to:
```suggestion
    if not utils_package.package_install.(session, "qemu-kvm")
        test.fail("qemu-kvm is not installed in the guest vm.")
```

Reply:
Updated.